libs/@guardian/newsletter-types/package.json

```diff
@@ -4,9 +4,14 @@
 	"private": false,
 	"description": "A set of types describing the output of the Guardian newsletters API",
 	"sideEffects": false,
+	"scripts": {
+		"fix": "pnpm lint --fix",
+		"lint": "eslint --cache ."
+	},
 	"devDependencies": {
 		"tslib": "2.6.2",
-		"typescript": "5.3.3"
+		"typescript": "5.3.3",
+		"wireit": "0.14.4"
```

Comment:
Did `wireit` become a dependency of this package?

Reply:
oop, not yet, thanks